CHANGES.md

```diff
@@ -677,6 +701,13 @@ OpenSSL 3.0
    time.  Instead applications should use L<EVP_DigestSignInit_ex(3)>,
    L<EVP_DigestSignUpdate(3)> and L<EVP_DigestSignFinal(3)>.
 
+   Finaly functions that assign or obtain DH objects from an EVP_PKEY such as
+   `EVP_PKEY_assign_DSA()`, `EVP_PKEY_get0_DSA()`, `EVP_PKEY_get1_DSA()`, and
+   `EVP_PKEY_set1_DSA()` are also deprecated.
+   Applications should instead either read or write an
+   EVP_PKEY directly using the OSSL_DECODER and OSSL_ENCODER APIs.
+   Or load an EVP_PKEY directly from DSA data using `EVP_PKEY_fromdata()`.
```

Comment:
from DSA?

Reply:
I'm more noticing the construction "... and OSSL_ENCODER APIs.  Or load ..."
I'd rather write that like this: "... and OSSL_ENCODER APIs, or load ..."
(in my writer upbringing, starting a sentence with "Or", "And" etc is generally frowned upon)

Reply:
I made the changes suggested by @levitte. I'm not sure what you wanted me to change @kroeckx? This reads ok to me?

Reply:
For some reason I didn't see the word data after it. I guess I would use "create an EVP_PKEY" instead of "load", but either is fine for me.